cluster/clientbin.sh

```diff
@@ -68,7 +68,7 @@ esac
 # Get the absolute path of the directory component of a file, i.e. the
 # absolute path of the dirname of $1.
 get_absolute_dirname() {
-  echo "$(cd "$(dirname "$1")" && pwd)"
+  (cd "$(dirname "$1")" && pwd)
```

Comment:
this is better than the prior code, but noting that this code is also subtly wrong FWIW. if "$1" is not a path the CD will fail and we'll return pwd == pwd of the caller

Reply:
In the end, I found that the function `get_absolute_dirname` had never been used. :rofl: 
Remove it safely.